core/appconfig/debug_container.dot

```diff
@@ -0,0 +1,3 @@
+digraph "" {
```

Comment:
Should we add a .gitignore entry?

Reply:
:+1:

Reply:
all .dot files? or only this one.

Reply:
Only this one. There are other .dot files already committed